src/ventus/wm/window.js

```diff
@@ -432,7 +440,10 @@ function(Emitter, Promise, View, WindowTemplate) {
 			// Open animation
 			this.el.show();
 			this.el.addClass('opening');
-			this.el.onAnimationEnd(function(){
+			if (this.animations) {
```

Comment:
Just one last thing. Do this only once in the constructor so that the window always have that class when the option is enabled... A plus would be to create the setter and getter for the "animations" and do it there.

Thanks a lot for implementing this! It is really helpful.

Reply:
You are right sir. Let me fix this.

Reply:
Something like this?